pkg/controller/volume/persistentvolume/framework_test.go

```diff
@@ -595,22 +599,62 @@ func newTestController(kubeClient clientset.Interface, volumeSource, claimSource
 	if classSource == nil {
 		classSource = fcache.NewFakeControllerSource()
 	}
-
+	// Pass in shared informers as usual but do not start them. Instead,
```

Comment:
The fake clientset supports "prepopulating" data as well as custom reactors. You should not need to be writing your own controllers in unit tests when using shared informers. How hard would it be do move to this model?